core/src/main/java/org/apache/carbondata/core/mutate/data/BlockMappingVO.java

```diff
@@ -30,6 +30,8 @@
 
   private Map<String, RowCountDetailsVO> completeBlockRowDetailVO;
 
+  private Map<String, String> blockToSegmentMapping;
```

Comment:
There are too many mappings in this class, can you avoid it. Its meaning is not easy to understand

Reply:
This map will help us to finding the segment id from the block path. I will add proper comments to this map in code. Can you suggest me any alternative to get the segment Id, Thanks